openedx/core/djangoapps/discussion_common/comment_client/thread.py

```diff
@@ -159,8 +164,6 @@ def _retrieve(self, *args, **kwargs):
     def flagAbuse(self, user, voteable):
         if voteable.type == 'thread':
             url = _url_for_flag_abuse_thread(voteable.id)
-        elif voteable.type == 'comment':
-            url = _url_for_flag_comment(voteable.id)
```

Comment:
Why was this removed?

Reply:
pylint complained that `_url_for_flag_comment` isn't defined.  After a code search, I verified that it's not defined anywhere.